src/All.php

```diff
@@ -0,0 +1,155 @@
+<?php namespace Leean\Acf;
+
+/**
+ * Class to provide helpers for getting a all ACF fields for an entity.
+ */
+class All
+{
+	/**
+	 * Is the ACF plugin active.
+	 *
+	 * @return bool
+	 */
+	public static function is_active() {
+		return function_exists( 'get_field_object' );
+	}
+
+	/**
+	 * Apply the default transforms if any exist for this field type.
+	 *
+	 * @param array $field The field.
+	 * @return mixed
+	 */
+	private static function apply_default_transform( $field ) {
+		$class = '\\' . __NAMESPACE__ . '\\Transforms\\' . str_replace( '_', '', ucwords( $field['type'], '_' ) );
```

Comment:
Can we create a method for this or a descriptive variable name since I'm not sure what the line is doing just by reading the line.